homeassistant/components/rainforest_eagle/data.py

```diff
@@ -59,8 +59,15 @@ async def async_get_type(hass, cloud_id, install_code, host):
         meters = None
 
     if meters is not None:
+        meter_idx = 0
+        # pylint: disable=unused-variable
+        for idx, meter in enumerate(meters):
+            if meters[idx].connection_status == "Connected":
```

Comment:
```suggestion
            if meter.connection_status == "Connected":
```

Reply:
Made the suggested change, and I was then able to remove the ```#pylint: disable=unused-variable``` statement! 👍